src/pages/Events/components/EventsDefaultView.tsx

```diff
@@ -0,0 +1,159 @@
+  const { event } = useAnalytics();
+  const getInitialFilters = useCallback((): Filters => {
+    const params = new URLSearchParams(location.search);
+    const expired = params.get('expired') ? Boolean(params.get('expired') === 'true') : false;
+    const open_for_sign_up = params.get('open_for_sign_up') ? Boolean(params.get('open_for_sign_up') === 'true') : undefined;
+    const category = params.get('category') || undefined;
+    const search = params.get('search') || undefined;
+    return { expired, category, search, open_for_sign_up };
+  }, []);
+  const { classes } = useStyles();
+  const navigate = useNavigate();
+  const lgDown = useMediaQuery((theme: Theme) => theme.breakpoints.down('lg'));
+  const { data: categories = [] } = useCategories();
+  const [filters, setFilters] = useState<Filters>(getInitialFilters());
+  const { data, error, hasNextPage, fetchNextPage, isLoading, isFetching } = useEvents(filters);
+  const events = useMemo(() => (data ? data.pages.map((page) => page.results).flat() : []), [data]);
+  const { register, control, handleSubmit, setValue, formState } = useForm<Filters>({ defaultValues: getInitialFilters() });
+  const isEmpty = useMemo(() => (data !== undefined ? !data.pages.some((page) => Boolean(page.results.length)) : false), [data]);
+
+  const resetFilters = () => {
+    setValue('category', '');
+    setValue('search', '');
+    setValue('expired', false);
+    setFilters({ expired: false, open_for_sign_up: false });
+    navigate(`${location.pathname}${argsToParams({ expired: false })}`, { replace: true });
+  };
+
+  const search = (data: Filters) => {
+    event('search', 'events', JSON.stringify(data));
+    setFilters(data);
+    navigate(`${location.pathname}${argsToParams(data)}`, { replace: true });
+    !lgDown || setSearchFormExpanded((prev) => !prev);
+  };
+
+  const [searchFormExpanded, setSearchFormExpanded] = useState(false);
+
+  const SearchForm = () => (
+    <form onSubmit={handleSubmit(search)}>
+      <TextField disabled={isFetching} formState={formState} label='Søk' margin='none' {...register('search')} />
+      {Boolean(categories.length) && (
+        <Select control={control} formState={formState} label='Kategori' name='category'>
+          {categories.map((value, index) => (
+            <MenuItem key={index} value={value.id}>
+              {value.text}
+            </MenuItem>
+          ))}
+        </Select>
+      )}
+      <Bool control={control} formState={formState} label='Tidligere' name='expired' type='switch' />
+      <Bool control={control} formState={formState} label='Kun med åpen påmelding' name='open_for_sign_up' type='switch' />
+      <SubmitButton disabled={isFetching} formState={formState}>
+        Søk
+      </SubmitButton>
+      <Divider sx={{ my: 1 }} />
+      <Button color='error' fullWidth onClick={resetFilters} variant='outlined'>
+        Tilbakestill
+      </Button>
+    </form>
+  );
+
+  return (
+    <Page>
```

Comment:
Her blir det Page inne i en annen Page, unngå det
```suggestion
    <>
```